modAionImpl/test/org/aion/zero/impl/cli/CliTest.java

```diff
@@ -102,6 +102,11 @@
     private static final File mainnetGenesis = new File(MAIN_CONFIG_PATH, genesisFileName);
     private static final File testnetGenesis = new File(TEST_CONFIG_PATH, genesisFileName);
 
+    private static final File fork = new File(TEST_RESOURCE_DIR, forkFileName);
+    private static final File oldFork = new File(CONFIG_PATH, forkFileName);
```

Comment:
need a test for behavior with old kernels config using the variable above

Reply:
disable to support the old config compatibility for the fork. And will put it in the release note.